parlai/tasks/task_list.py

```diff
@@ -411,4 +411,11 @@
         "tags": ["All", "Visual"],
         "description": "COCO annotations derived from the 2015 COCO Caption Competition. Link to dataset: http://cocodataset.org/#download",
     },
+    {
+        "id": "ConvAI2_wild_evaluation",
+        "display_name": "ConvAI2_wild_evaluation",
+        "task": "convai2_wild_evaluation",
+        "tags": ["All",  "ChitChat" ],
+        "description": "Dataset collected during the wild evaluation of ConvaAI2 participants bots (http://lnsigo.mipt.ru/export/datasets/convai/convai2_wild_evaluation_0.2.tgz). 60% train, 20% valid and 20% test is chosen at random from the whole dataset."
```

Comment:
best not to include the tgz here as it will likely be updated, but could include http://convai.io perhaps?

Reply:
jaseweston, shure, fixed it.